mne_bids/write.py

```diff
@@ -300,20 +301,25 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False,
             if isinstance(meas_date, datetime):
                 meas_datetime = meas_date
             else:
-                meas_datetime = datetime.fromtimestamp(meas_date)
+                meas_datetime = datetime.fromtimestamp(meas_date, tz=timezone.utc)
             subject_age = _age_on_date(bday, meas_datetime)
         else:
             subject_age = "n/a"
 
-    data.update({'age': [subject_age], 'sex': [sex]})
+        # add handedness
+        hand_options = {0: 'n/a', 1: 'R', 2: 'L', 3: 'A'}  # not according to mne conventions (only r/l)
+        hand = hand_options[subject_info.get('hand', 0)]
```

Comment:
for readability reasons I would move this to 291 right below the sex variable declaration (with a blank line in between).

then we would have two short extractions, and then a long one.

Also: What do you mean by the comment "not according to mne conventions (only r/l)"?

Reply:
i rearranged the order..

concerning the comment: 
the mne [documentation](https://mne.tools/dev/generated/mne.Info.html#mne.Info) suggests only two options: 
`hand : int`
`Handedness (1=right, 2=left).`

no n/a, no ambidextrous. 
this was more of a reminder for myself to open an issue on mne's git

Reply:
I see, in that case I would prefer

```
        # XXX: MNE currently only handles R/L, follow <insert MNE issue ID here>
        hand_options = {0: 'n/a', 1: 'R', 2: 'L', 3: 'A'}
```

and if you don't want to make an issue now, don't put <insert MNE issue ID here>

Reply:
i followed your suggestion and included a link to the issue